pages/blog/team-intro/index.mdx

```diff
@@ -0,0 +1,66 @@
+    emerging fields of DesignOps and FrontEndOps and their career aspirations.
+
+---
+
+
+*Hey there! Today, I’m chatting with Varya, Irina, and Alex, three design-system
+consultants from Bridge the Gap agency. They're going to share their views on
+the role of design systems in modern software development. Let's start with some
+introductions. Varya, would you like to begin?*
+
+![](./team.jpg)
+
+Photo by <a href="https://unsplash.com/@brookecagle?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Brooke Cagle</a> on <a href="https://unsplash.com/s/photos/it-team-women?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+**Varya:** Hi everyone! My name is Varya, and I'm a design systems and front-end architecture consultant with years of experience. I'm currently working with the team you're meeting today.
+
+**Irina:** Hi, I'm Irina, a front-end developer with a focus on React/TypeScript stack. I love exploring new tech and currently working on a large enterprise application while also consulting on design systems.
+
+**Alex:** Hey, I'm Alex, a front-end developer who primarily works with React and JS. I'm learning Web Components now.
+
+*So, where are you all based?*
+
+**Varya:** I'm from Espoo, Finland.
+
+**Irina:** I'm from Greece.
+
+**Alex:** I'm from Vantaa, Finland.
+
+*Interesting! How did you all end up on the same team?*
+
+**Irina:** I was Varya's intern a few years back, and we worked together on some amazing projects. We continue to collaborate to this day, even doing design system workshops together.
+
+**Alex:** I stumbled upon Varya's internship on Twitter, and my husband sent me the same link from his IT-chat. After an interview and some waiting, I was offered the job.
+
+**Varya:** As a solo consultant, I was on the lookout for people to join me on projects. When I met Irina and Alex, I knew they would be a great fit, and so far, we make an awesome team. We work remotely, which is a great bonus thanks to the pandemic.
+
+*What's your team's focus right now?*
+
+**Varya:** We are working on design systems projects in React and Web Components technologies. But we're open to exploring new things.
+
+**Irina:** Design systems are what I love doing the most. I want to grow our team and work on exciting new projects.
+
+**Varya:** For me, bridging design and development is what we excel at. Our team is great at managing this balance, and I believe we can take it to the next level with DesignOps and FrontEndOps.
+
+*What are those?*
+
+**Varya:** DesignOps and FrontEndOps are processes that focus on optimizing the pipelines and scripts we write for projects. While DevOps is prevalent, we think FrontEndOps or DesignSystemOps are specialized fields that require special attention.
+
+**Irina:** I agree. There is a unique set of challenges when dealing with design systems that need attention.
+
+Alex, what are your career aspirations?
```

Comment:
@varya The question and text below are redundant. I think it's better to leave only a discussion about Design System.